tests/MassTransit.MartenIntegration.Tests/TwoSaga_Specs.cs

```diff
@@ -77,6 +85,12 @@ protected override void ConfigureInMemoryReceiveEndpoint(IInMemoryReceiveEndpoin
                 configurator.ConfigureSaga<TestInstance>(busRegistrationContext);
                 configurator.ConfigureSaga<TestInstance2>(busRegistrationContext);
             }
+
+            private Task EnsureMartenHostedServiceIsStarted() =>
```

Comment:
We have no AppHost here, its just a ServiceCollection.
Need to ensure this service starts, which is automatically registered as an IHostedService via the 'ApplyAllDatabaseChangesOnStartup' extension method.
Lets just resolve it, and run it